<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tracking</class>
 <widget class="QMainWindow" name="tracking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1875</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tracking</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="stop_stat_view1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>750</y>
      <width>355</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="track_stat_view1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>750</y>
      <width>140</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="screenshot_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/screenshot.jpg</normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/screenshot.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="wechat_push_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>760</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/wechat_push.png</normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/wechat_push.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_line_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>760</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/connect_lin.png</normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/connect_lin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="analyze_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>760</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/analyze_point.jpg</normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/analyze_point.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_interface">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>760</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/clear1.png</normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/clear1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="window_quit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>760</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/exit.jpg</normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/exit.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="screen_label">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>800</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>截图</string>
    </property>
   </widget>
   <widget class="QLabel" name="wechat_label">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>800</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>登录微信</string>
    </property>
   </widget>
   <widget class="QLabel" name="conn_label">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>800</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>连接轨迹</string>
    </property>
   </widget>
   <widget class="QLabel" name="ana_label">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>800</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>轨迹分析</string>
    </property>
   </widget>
   <widget class="QLabel" name="clear_label">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>800</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>清除界面</string>
    </property>
   </widget>
   <widget class="QLabel" name="exit_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>800</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="txt_clear_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>760</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/txt_clear.jpg</normaloff>../build-tracking-Desktop_Qt_5_9_0_MSVC2013_64bit-Debug/txt_clear.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="txtclr_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>800</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>清空坐标</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mymenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1875</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="ScreenShot"/>
    <addaction name="separator"/>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="tool">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="WechatPush"/>
    <addaction name="TrackAnalyze"/>
    <addaction name="TrackConnect"/>
    <addaction name="InterfaceClear"/>
    <addaction name="txt_clear"/>
    <addaction name="rece_data"/>
   </widget>
   <addaction name="file"/>
   <addaction name="tool"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="ScreenShot">
   <property name="text">
    <string>截图</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="WechatPush">
   <property name="text">
    <string>打开微信通知</string>
   </property>
  </action>
  <action name="TrackAnalyze">
   <property name="text">
    <string>分析轨迹</string>
   </property>
  </action>
  <action name="TrackConnect">
   <property name="text">
    <string>轨迹连线</string>
   </property>
  </action>
  <action name="InterfaceClear">
   <property name="text">
    <string>清空界面</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="txt_clear">
   <property name="text">
    <string>清空txt</string>
   </property>
  </action>
  <action name="rece_data">
   <property name="text">
    <string>开始接受数据</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
